runtime/doc/lua.txt

```diff
@@ -2428,8 +2428,27 @@ vim.loader.reset({path})                                  *vim.loader.reset()*
 ==============================================================================
 Lua module: vim.uri                                                  *vim.uri*
 
+vim.uri_decode({str})                                       *vim.uri_decode()*
+    URI-decodes a string containing percent escapes.
+
+    Parameters: ~
+      • {str}  (string) string to decode
+
+    Return: ~
+        (string) decoded string
+
+vim.uri_encode({str}, {rfc})                                *vim.uri_encode()*
```

Comment:
Before 0.10 release we should move all these to a `vim.uri` module. 

The top-level `uri_` functions are my fault but we don't want to continue adding more. Because we need a `Uri` object too. That's out of scope for this PR, but just noting for reference.